src/routes/auth/auth.route.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import UsersController from '../../controllers/signup.controller';
+import loginHandler from '../../controllers/login.controller';
+import loginMiddleware from '../../middlewares/login.middleware';
```

Comment:
Create a single file for all  auth middlewares and change(Rename) here after creating it.